### ISSUE TITLE
Blocks: File block should have low priority on its `files` transform

### ISSUE THREAD

**Issue by aduth** [WordPress/gutenberg#8022]
Related: https://github.com/WordPress/gutenberg/pull/7622/files#r203394758

As implemented, the File block `files` transform has identical priority as any other file transform, when in fact this should be the fallback. For example, dropping an image should take advantage of the Image block's `files` transform before this one. I don't know if this is currently working, and if it is it only incidental and prone to future breakage.

See: https://wordpress.org/gutenberg/handbook/block-api/#transforms-optional

>To control the priority with which a transform is applied, define a priority numeric property on your transform object, where a lower value will take precedence over higher values. This behaves much like a WordPress hook. Like hooks, the default priority is 10 when not otherwise set.
